test/e2e/propagationpolicy_test.go

```diff
@@ -1048,6 +1050,74 @@ var _ = ginkgo.Describe("[AdvancedPropagation] propagation testing", func() {
 		})
 	})
 
+	ginkgo.Context("Delete the propagationPolicy", func() {
```

Comment:
`Delete the propagationPolicy and label checking` would be better?

Reply:
`ginkgo.Context` can contain a test group. I think the description here does not need to include label checking. It represents the test case of the deletion action. If there are other test cases for deletion later, it can be added here.